servers/rendering/renderer_rd/shaders/forward_mobile/scene_forward_mobile.glsl

```diff
@@ -105,10 +105,6 @@ layout(location = 6) mediump out vec3 binormal_interp;
 layout(location = 7) highp out vec4 diffuse_light_interp;
 layout(location = 8) highp out vec4 specular_light_interp;
 
-layout(constant_id = 9) const bool sc_disable_omni_lights = false;
```

Comment:
This cant have worked since the uber shader work was done to create methods to get these state values, to prove I had not broken USE_VERTEX_LIGHTING's directional shadow sampling logic I had to fix this part, I am unware of a defect item for this.